client/src/pages/ChatAssistancePage.js

```diff
@@ -1,11 +1,134 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { Layout, Input, Button, Card, Typography, Space } from 'antd';
+import { SendOutlined, RobotOutlined, UserOutlined } from '@ant-design/icons';
+import 'antd/dist/reset.css';
+
+const { Header, Content } = Layout;
+const { Text } = Typography;
 
 function ChatAssistantPage() {
```

Comment:
can you use arrow operators instead of functions. They are the latest ones and can be used